icons/ruler.json

```diff
@@ -8,9 +8,13 @@
   ],
   "tags": [
     "measure",
+    "centimetres",
     "meter",
     "foot",
-    "inch"
+    "feet",
+    "inches",
+    "units",
+    "stationery"
```

Comment:
Maybe this (and also the pencil one?) could have some more tags, i.e. length, width, depth, breadth, dimensions, size, extent, measurements.